### DISCUSSION TITLE
Discourse workarounds

### DISCUSSION THREAD

**Discussion by bbbhltz** [qutebrowser/qutebrowser#7830] [Q&A]
I didn't want to reopen #7572 but I've encountered this problem again. The solution [here](https://github.com/qutebrowser/qutebrowser/issues/7572#issuecomment-1590037971) works to view and navigate the forum, but I cannot reply or create topics.

Are there any other workarounds to this or should I move on to qt6?

**Comment by The-Compiler** [comment]
You should move to Qt 6 if you can. The v3.0.0 release with it is [around the corner](https://github.com/qutebrowser/qutebrowser/milestones/v3.0.0).
